zou/app/blueprints/source/csv/assets.py

```diff
@@ -52,14 +56,83 @@ def prepare_import(self, project_id):
             .all()
         }
 
+    def get_tasks_update(self, row):
+        tasks_update = []
+        for task_type in self.task_types_in_project_for_assets:
+            task_status_name = row.get(task_type.name, None)
+            task_status_id = None
+            if task_status_name is not None:
+                for status_id, status_names in self.task_statuses.items():
+                    if task_status_name in status_names:
+                        task_status_id = status_id
+                        break
+                if task_status_id is None:
+                    raise TaskStatusNotFoundException(
```

Comment:
Be careful, raising exceptions like this generates a 404 error.

Reply:
Effectively, I have to use ImportRowException